### DISCUSSION TITLE
Nextjs 13 in app directory, it does not render tailwind css

### DISCUSSION THREAD

**Discussion by Tarbez** [vercel/next.js#42105] [Help] [answered]
I just started using NEXTjs 13 and I have upgraded my tailwind config file to make sure that it reads the css but it is still not reading any tailwind css when the file is inside the app directory. 

``` -tailwind.config.js file
 content: [
    "./app/**/*.{js,ts,jsx,tsx}",
    "./pages/**/*.{js,ts,jsx,tsx}",
    "./components/**/*.{js,ts,jsx,tsx}",
  ],
```

If I use pages, it will render normally, what am I missing here?

**Comment by santhaug** [comment, marked as answer]
Have you imported the global.css file in the root layout (app/layout.tsx)?

https://beta.nextjs.org/docs/styling/tailwind-css

**Reply by Tarbez** [reply]
Yes, after I submitted this post, I kept on checking what I was missing and it seemed like the global.css was missing and I couldn't remove this post. My bad, stupid mistake.
